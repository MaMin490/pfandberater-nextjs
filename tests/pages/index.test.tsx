import React from 'react';
import { shallow } from 'enzyme';

import Home from '../../pages/index';

describe('Home', () => {
  it('should render correctly', () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
