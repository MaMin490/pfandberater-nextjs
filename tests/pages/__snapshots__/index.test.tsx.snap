// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<Fragment>
  <styled.div>
    <styled.h1>
      willkommen
    </styled.h1>
    <styled.h2>
      so funktionierts
    </styled.h2>
    <ForwardRef
      autoplay={true}
    >
      <div>
        <styled.h3>
          1
        </styled.h3>
      </div>
      <div>
        <styled.h3>
          2
        </styled.h3>
      </div>
      <div>
        <styled.h3>
          3
        </styled.h3>
      </div>
    </ForwardRef>
    <styled.div>
      <styled.button>
        sei dabei
      </styled.button>
    </styled.div>
  </styled.div>
</Fragment>
`;
